docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,143 @@
+            "float32": 1.2,
+            "float64": 1.2,
+            "int": 1,
+            "int32": 1,
+            "int64": 1,
+            "string": "orange"
+        }
+    }
+}
+```
+
+:::note
+`.Position`, `.Key`, and `.Payload.Before` are represented as `Base64` encoded in the example above because these will be a byte slice when represented as JSON.
+:::
+
+## Metadata fields
+
+As part of an OpenCDC record, there will be a set of fields provided that will vary depending on the connector. These fields can be common to all **OpenCDC** records as part of our standard, some related to **Conduit**, and others that will be provided by each **Connector** implementation independently. These fields can be useful to define conventions that will be then used by Conduit to expand its functionality. Notice that all these fields use a dot notation syntax to indicate what they refer to, preventing accidental clashes. Here are the ones you can find:
+
+### OpenCDC
+
+- `opencdc.createdAt` can contain the time when the record was created in the 3rd party system. The expected format is a Unix timestamp in nanoseconds.
+- `opencdc.readAt` can contain the time when the record was read from the 3rd party system. The expected format is a Unix timestamp in nanoseconds.
+- `opencdc.version` contains the version of the OpenCDC format (e.g., "v1"). This field exists to ensure the OpenCDC format version can be easily identified in case the record gets marshaled into a different untyped format (e.g. JSON).
+
+```json
+{
+    ...
+    "metadata": "{
+        "opencdc.createdAt": "1663858188836816000",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+### Conduit
+
+Only available in records once they are read by a **source plugin**:
+
+- `conduit.source.plugin.name` is the name of the source plugin that created the record.
+- `conduit.source.plugin.version` is the version of the source plugin that created the record.
+- `conduit.source.connector.id` is the ID of the source connector that received the record.
+
+```json
+{
+    ...
+    "metadata": "{
+        "conduit.source.connector.id": "connectorID",
+        "conduit.source.plugin.name": "example",
+        "conduit.source.plugin.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+Only available in records once they are written by a **destination plugin**:
+
+- `conduit.destination.plugin.name` is the name of the destination plugin that has written the record.
+- `conduit.destination.plugin.version` is the version of the destination plugin that has written the record.
+
+```json
+{
+    ...
+    "metadata": "{
+        "conduit.destination.plugin.name": "example",
+        "conduit.destination.plugin.version": "v1",
+        ...
+    }",
+    ...
+}
+```
+
+When a record is sent to the [Dead-Letter Queue (DLQ)](/dead-letter-queue), you'll also see these extra fields that will give you an insight into why the landed in the DQL.
```

Comment:
You need to stop dropping records, this is breaking our record delivery guarantees!
```suggestion
When a record is sent to the [Dead-Letter Queue (DLQ)](/dead-letter-queue), you'll also see these extra fields that will give you an insight into why the record landed in the DLQ.
```